### DISCUSSION TITLE
Light status update on Google Home app

### DISCUSSION THREAD

**Discussion by umibozu83** [andrei-tatar/node-red-contrib-smartnora#316] [Q&A]
I noticed that if, for example, I turn on a light via voice command on the Google Home app, the status updates. if I do it with the home assistant interface it doesn't update the status. For simple switching on and off I was able to update the On Off status with a "trigger: State" node. What if I also want to report the brightness value of the light?

**Comment by andrei-tatar** [comment]
@umibozu83 If you want to update the status of a light back to Google Home, you need to provide the new light state as an input to the light node.

For example two inject nodes that inject the value on/off to the NORA light.
![image](https://github.com/user-attachments/assets/19417d56-6333-46b4-92f2-ebeb66afae3e)


I'm not using home assistant but I suspect you need to get the light status from Home assistant injected into the NORA light node.
